tests/io/test_partitioned_dataset.py

```diff
@@ -401,7 +401,7 @@ def test_dataset_creds(self, pds_config, expected_ds_creds, global_creds):
         assert pds._credentials == global_creds
 
 
-BUCKET_NAME = "fake_bucket_name"
+BUCKET_NAME = "s3_fake_bucket_name"
```

Comment:
Wouldn't it be better to cover both cases (the one before the fix and the one after)? Just to make sure 🤞🏻

Reply:
Wdyt @noklam ?

Reply:
I will make a follow up PR on this.